qgis_deployment_toolbelt/jobs/job_shortcuts.py

```diff
@@ -147,10 +147,23 @@ def run(self) -> None:
             raise NotImplementedError
 
     # -- INTERNAL LOGIC ------------------------------------------------------
+    def get_qgis_path(self, qgis: str) -> Union[Path, None]:
+        """Try to get qgis path.
+
+        :param str qgis: qgis path as mentioned into the scenario file
+        :return Union[Path, None]: _description_
```

Comment:
Docstring is incomplete

Reply:
Sorry, bad copy and paste. The docstring of the `get_icon_path` function function was incomplete on the main branch. I added a description for the 2 functions.